upgrade_downgrade/test_dual_control_plane_upgrade_downgrade_boutique.sh

```diff
@@ -123,8 +123,11 @@ kubectl label namespace "${TEST_NAMESPACE}" istio-injection- || echo "istio-inje
 kubectl label namespace "${TEST_NAMESPACE}" istio.io/rev="${FROM_REVISION}"
 
 function deploy_and_wait_for_services() {
-  local -n services="$1"
-  for svc in "${services[@]}"; do
+  # Do not name this services. We'll end up with a situation
```

Comment:
nit: Add `Warning: ` in the beginning of this comments

Reply:
Done. Added the warning

Actually it is easy to run into this issue. It was caused by this
```bash
function deploy_and_wait_for_services() {
    local -n services="$1"
    # ... rest of the stuffs
}
# Calling with services as parameter expanded to
# local -n services=services and caused the error
deploy_and_wait_for_services services
```